doc/src/manual/environment-variables.md

```diff
@@ -0,0 +1,327 @@
+
+### `JULIA_GC_ALLOC_POOL`, `JULIA_GC_ALLOC_OTHER`, `JULIA_GC_ALLOC_PRINT`
+
+If set, these environment variables take strings that optionally start with the
+character `'r'`, followed by a string interpolation of a colon-separated list of
+three signed 64-bit integers (`int64_t`). This triple of integers `a:b:c`
+represents the arithmetic sequence `a`, `a + b`, `a + 2*b`, ... `c`.
+
+*   If it's the `n`th time that `jl_gc_pool_alloc()` has been called, and `n`
+    belongs to the arithmetic sequence represented by `$JULIA_GC_ALLOC_POOL`,
+    then garbage collection is forced.
+*   If it's the `n`th time that `maybe_collect()` has been called, and `n` belongs
+    to the arithmetic sequence represented by `$JULIA_GC_ALLOC_OTHER`, then garbage
+    collection is forced.
+*   If it's the `n`th time that `jl_gc_collect()` has been called, and `n` belongs
+    to the arithmetic sequence represented by `$JULIA_GC_ALLOC_PRINT`, then counts
+    for the number of calls to `jl_gc_pool_alloc()` and `maybe_collect()` are
+    printed.
+
+If the value of the environment variable begins with the character `'r'`, then
+the interval between garbage collection events is randomized.
+
+!!! note
+
+    These environment variables only have an effect if Julia was compiled with
+    garbage-collection debugging (that is, if `WITH_GC_DEBUG_ENV` is set to `1`
+    in the build configuration).
+
+### `JULIA_GC_NO_GENERATIONAL`
+
+If set to anything besides `0`, then the Julia garbage collector never performs
+"quick sweeps" of memory.
+
+!!! note
+
+    This environment variable only has an effect if Julia was compiled with
+    garbage-collection debugging (that is, if `WITH_GC_DEBUG_ENV` is set to `1`
+    in the build configuration).
+
+### `JULIA_GC_WAIT_FOR_DEBUGGER`
+
+If set to anything besides `0`, then the Julia garbage collector will wait for
+the debugger to attach instead of aborting whenever there's a critical error.
+
+!!! note
+
+    This environment variable only has an effect if Julia was compiled with
+    garbage-collection debugging (that is, if `WITH_GC_DEBUG_ENV` is set to `1`
+    in the build configuration).
+
+### `ENABLE_JITPROFILING`
+
+If set to anything besides `0`, then the compiler will create and register an
+event listener for just-in-time (JIT) profiling.
+
+!!! note
+
+    This environment variable only has an effect if Julia was compiled with JIT
+    profiling support, using either
+
+*   Intel's [VTune™ Amplifier](https://software.intel.com/en-us/intel-vtune-amplifier-xe)
+    (`USE_INTEL_JITEVENTS` set to `1` in the build configuration), or
+*   [OProfile](http://oprofile.sourceforge.net/news/) (`USE_OPROFILE_JITEVENTS` set to `1`
+    in the build configuration).
+
+### `JULIA_LLVM_ARGS`
+
+Arguments to be passed to the LLVM backend.
+
+!!! note
+
+    This environment variable has an effect only if Julia was compiled with
+    `JL_DEBUG_BUILD` set—in particular, the `julia-debug` executable is always
```

Comment:
"set - in particular" https://github.com/JuliaLang/julia/pull/20162#discussion_r98843854